libs/libzauth/libzauth-c/src/lib.rs

```diff
@@ -284,8 +312,191 @@ impl From<TokenVerification> for ZauthTokenVerification {
     fn from(t: TokenVerification) -> ZauthTokenVerification {
         match t {
             TokenVerification::Verified => ZauthTokenVerification::Verified,
-            TokenVerification::Invalid  => ZauthTokenVerification::Invalid,
-            TokenVerification::Pending  => ZauthTokenVerification::Pending,
+            TokenVerification::Invalid => ZauthTokenVerification::Invalid,
+            TokenVerification::Pending => ZauthTokenVerification::Pending,
+        }
+    }
+}
+
+// ------------------------------------------------------------------------
+// OAuth
+
+#[repr(C)]
+#[derive(Clone, Copy, Debug)]
+pub enum OAuthResultStatus {
+    Ok = 0,
+    InsufficientScope = 1,
+    NullArg = 2,
+    IoError = 3,
+    Utf8Error = 4,
+    Panic = 99,
+}
+
+pub struct OAuthJwk(String);
+
+#[repr(C)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
You probably don't want to derive Copy here. It's basically an "implicit" clone. And since your struct holds pointer, it could silently copy those pointers and somehow result in double free errors